tutorials/plot_ecog.py

```diff
@@ -0,0 +1,47 @@
+"""
+======================
+Working with ECoG data
+======================
```

Comment:
this page needs to be integrated in the tutorials page:

http://martinos.org/mne/stable/tutorials.html